lib/fluent/plugin/out_typecast.rb

```diff
@@ -17,7 +17,7 @@ class TypecastOutput < Output
   config_param :tag,         :string, default: nil
   config_param :prefix,      :string, default: nil
 
-  ITEM_TYPES = ['string', 'integer', 'float', 'bool', 'time', 'array']
+  ITEM_TYPES = ['json', 'string', 'integer', 'float', 'bool', 'time', 'array']
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.